src/posthog-persistence.js

```diff
@@ -212,8 +202,8 @@ PostHogPersistence.prototype.update_referrer_info = function (referrer) {
 
 PostHogPersistence.prototype.get_referrer_info = function () {
     return _.strip_empty_properties({
-        $initial_referrer: this['props']['$initial_referrer'],
-        $initial_referring_domain: this['props']['$initial_referring_domain'],
+        $referrer: this['props']['$referrer'],
+        $referring_domain: this['props']['$referring_domain'],
```

Comment:
actually wondering if this was needed given we already had referrer props working correctly but anyway

Reply:
it's a function in the module, can't someone be calling it? Otherwise we might be able to drop that.